packages/cloud/connection_details/tabs/api_keys_tab/views/configuration_form/configuration_form_controlled.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  EuiButton,
+  EuiCallOut,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiSpacer,
+} from '@elastic/eui';
+import * as React from 'react';
+import { i18n } from '@kbn/i18n';
+import { ManageKeysLink } from '../../components/manage_keys_link';
+
+export interface ConfigurationFormControlledProps {
+  name: string;
+  loading?: boolean;
+  error?: Error | unknown;
+  onNameChange: React.ChangeEventHandler<HTMLInputElement>;
+  onSubmit: React.FormEventHandler<HTMLFormElement>;
+}
+
+export const ConfigurationFormControlled: React.FC<ConfigurationFormControlledProps> = ({
+  name,
+  loading,
+  error,
+  onNameChange,
+  onSubmit,
+}) => {
+  const body = (
+    <>
+      {!!error && (
+        <>
+          <EuiCallOut
+            color="danger"
+            iconType="error"
+            title={i18n.translate('cloud.connectionDetails.tab.apiKeys.error.title', {
+              defaultMessage: 'Error',
+            })}
+          >
+            {error instanceof Error ? error.message : String(error)}
+          </EuiCallOut>
+          <EuiSpacer />
+        </>
+      )}
+
+      <EuiFormRow
+        label={i18n.translate('cloud.connectionDetails.tab.apiKeys.nameField.label', {
+          defaultMessage: 'API key name',
+        })}
+        helpText={i18n.translate('cloud.connectionDetails.tab.apiKeys.nameField.helpText', {
+          defaultMessage: 'A good name makes it clear what your API key does.',
+        })}
+        error={'test'}
+        isDisabled={loading}
+        fullWidth
+      >
+        <EuiFieldText
+          name="api-key-name"
+          disabled={loading}
+          isLoading={loading}
+          value={name}
+          onChange={onNameChange}
+        />
+      </EuiFormRow>
+    </>
+  );
+
+  const footer = (
+    <EuiFlexGroup justifyContent="spaceBetween">
```

Comment:
The footer content is miss-aligned vertically 

![Screenshot 2024-04-17 at 14 07 48](https://github.com/elastic/kibana/assets/7784120/5adf2531-7031-4679-ac67-097585578cc3)

Reply:
It was aligned to the top :) Anyways, aligned it to the middle, both are fine IMO.